qu19041_ex3.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import matplotlib.pyplot as pyplot
+
+# PART (A)
+
+# Here we first define the function to be integrated followed by the function to evaluate the 
+# integral for conditions c.
+def integrand(x_screen, x_aperture, c):
+    return np.exp( c['waveno'] * 1j * (x_screen - x_aperture)**2 / (2*c['screendist']) )
```

Comment:
I don't bother with these spaces after the (. You're doing both styles in this file

Reply:
Also, try and be consistent about spaces around operators. Sometimes you have them, sometimes you don't.